update-db.js

```diff
@@ -158,4 +200,32 @@ module.exports = function updateDB (print) {
   childProcess.execSync(del + ' caniuse-lite')
 
   print('caniuse-lite has been successfully updated\n')
+
+  var currentBrowsersList
+  if (!browsersListRetrievalError) {
+    try {
+      currentBrowsersList = getBrowsersList()
+    } catch (e) /* istanbul ignore next */ {
+      browsersListRetrievalError = e
+    }
+  }
+
+  if (browsersListRetrievalError) {
+    print(browsersListRetrievalError.stack)
```

Comment:
I think error can be useful, e.g. we get error when npx is not installed, user can fix it based on error message

Reply:
Yeap. Good idea.